docs/assets/scss/hugo-uswds-custom.scss

```diff
@@ -18,3 +18,15 @@
     float: inherit;
   }
 }
+
+.usa-site-alert--info .usa-alert:before {
+    -webkit-mask: url(https://pages.nist.gov/oscal-tools/img/usa-icons/info.svg) no-repeat center
+    }
+
+.usa-banner__button {
+  @include at-media("tablet") {
+    &:after {
+    -webkit-mask:  url(http://localhost:1313/oscal-tools//img/usa-icons/expand_more.svg) no-repeat center
```

Comment:
I believe this URL should be: https://pages.nist.gov/oscal-tools/img/usa-icons/expand_more.svg . Can you please revisit it?